src/packages/migrate/src/utils/seed.ts

```diff
@@ -1,47 +1,165 @@
 import fs from 'fs'
 import path from 'path'
 import execa from 'execa'
-import resolvePkg from 'resolve-pkg'
 import hasYarn from 'has-yarn'
 import chalk from 'chalk'
-import globalDirectories from 'global-dirs'
 import pkgUp from 'pkg-up'
 import { promisify } from 'util'
+import { getPrismaConfigFromPackageJson, logger } from '@prisma/sdk'
+import Debug from '@prisma/debug'
 
+const debug = Debug('prisma:migrate:seed')
 const readFileAsync = promisify(fs.readFile)
 
-export function isPackageInstalledGlobally(
-  packageName: string,
-): 'npm' | 'yarn' | false {
-  try {
-    const usingGlobalYarn = fs.existsSync(
-      path.join(globalDirectories.yarn.packages, packageName),
+/*
+  Checks if user has a prisma/seed.ts or prisma/seed.js or prisma/seed.sh
+  If prisma.seed is not set in package.json it will return the best error message to help the user
+*/
+export async function verifySeedConfigAndReturnMessage(
+  schemaPath: string | null,
+): Promise<string | undefined> {
+  const cwd = process.cwd()
+
+  // Detect if seed files are next to prisma.schema file
+  const detected = detectSeedFiles(cwd, schemaPath)
+
+  const prismaConfig = await getPrismaConfigFromPackageJson(cwd)
+
+  // New config is set in the package.json, no need for an error message
+  if (prismaConfig && prismaConfig.data?.seed) {
+    return undefined
+  }
+
+  // If new "seed" config is not set, help user to set it
+  const packageManager = hasYarn() ? 'yarn add -D' : 'npm i -D'
+
+  // TODO link to docs
+  let message = `${chalk.red(
+    'To configure seeding in your project you need to add a "seed" property in your package.json with the command to execute it:',
+  )}
+
+1. Open the package.json of your project
+`
+
+  if (detected.numberOfSeedFiles === 1) {
+    // Probably was using seed before 2.27.0 and need to add the seed property in package.json
+    message += `2. Add the following example to it:`
+
+    if (detected.js) {
+      message += `
+\`\`\`
+"prisma": {
+  "seed": "node ${detected.js}"
+}
+\`\`\`
+`
+    } else if (detected.ts) {
+      message += `
+\`\`\`
+"prisma": {
+  "seed": "ts-node ${detected.ts}"
+}
+\`\`\`
+
+3. Install the required dependencies by running:
+${chalk.green(`${packageManager} ts-node typescript @types/node`)}
+`
+    } else if (detected.sh) {
+      message += `
+\`\`\`
+"prisma": {
+  "seed": "${detected.sh}"
+}
+\`\`\`
+And run \`chmod +x ${detected.sh}\` to make it executable.`
+    }
+  } else {
+    message += `2. Add one of the following example to your package.json:
```

Comment:
```suggestion
    message += `2. Add one of the following examples to your package.json:
```

But the same applies as commented before: https://github.com/prisma/prisma/pull/8146/files#r667486929

Reply:
Shouldn't this also mention that the `seed.x` file actually needs to be created?